### PR TITLE
Fix hierarchical commands

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Update common lib where a fix for hierarchical commands had been applied. Add help for new commands - '$ appbuilder device android' and '$ appbuilder device ios'

Fix is part of http://teampulse.telerik.com/view#item/283188

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
run ci

File: resources/help.txt [hunk 1]
```diff
@@ -295,17 +295,26 @@ the project retains any existing project configuration.
...
 Lists all recognized connected devices with serial number and index, grouped by platform. In this version of the Telerik AppBuilder CLI,
 you can connect only iOS and Android devices.
 
-When the target platform is Android, the Telerik AppBuilder CLI lists all connected physical and running Android virtual devices.
+<Command> is a related command that extends the device command. You can run the following related commands:
+    android - Lists all recognized connected Android physical and running Android virtual devices.
+    ios - Lists all recognized connected iOS devices.
+    log - Opens the device log stream for a selected connected device.
+    list-applications - Lists the installed applications on all connected Android or iOS devices.
+    run - Runs the selected application on a connected Android or iOS device.
```
Comment (teobugslayer) [inline resources/help.txt]: I think the "or iOS" part should be guarded in if (isMac) context.

File: resources/help.txt [hunk 2]
```diff
@@ -314,11 +323,18 @@ Options:
...
       after the specified time expires. If not set, the default value is 4000.
+--[/]--
 
-<Command> is a related command that extends the device command. You can run the following related commands:
-    log - Opens the device log stream for a selected connected device.
-    list-applications - Lists the installed applications on all connected Android or iOS devices.
-    run - Runs the selected application on a connected Android or iOS device.
+--[device|ios]--
+Usage:
+    $ appbuilder device ios [--timeout  <Milliseconds>]
```
Comment (teobugslayer) [inline resources/help.txt]: Extra space before Milliseconds

**Comment by teobugslayer** [comment]
Terrific :) :+1:

**Comment by teobugslayer** [comment]
ping @ikoevska for the content

**Pull request merged**